src/main/java/ir/sahab/nexus/plugin/tag/internal/TagRestResource.java

```diff
@@ -143,6 +146,16 @@ public Tag clone(@PathParam("name") String name, TagCloneRequest request) {
         return tagStore.cloneExisting(request.getSourceName(), name, request.getAppendingAttributes());
     }
 
+    @POST
+    @Path("/import_tags")
```

Comment:
Why not import-tags?
i think i url dash is better than underline. isn't it?

Reply:
Done